st2reactor/st2reactor/garbage_collector/base.py

```diff
@@ -222,3 +231,15 @@ def _purge_trigger_instances(self):
             LOG.exception('Failed to trigger instances: %s' % (str(e)))
 
         return True
+
+    def _fail_inquiries(self):
```

Comment:
Let's correct the name of this function as well (i.e. _cleanup_inquiries)

Reply:
Blech.....sorry, fixed in 4254ceb72